### ISSUE TITLE
failure in rc_mlx5/test_uct_peer_failure_multiple

### ISSUE THREAD

**Issue by yosefe** [openucx/ucx#1773]
http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4311/label=hpc-test-node3,worker=0/console
```
16:59:12 [----------] 1 test from rc_mlx5/test_uct_peer_failure_multiple
16:59:12 [ RUN      ] rc_mlx5/test_uct_peer_failure_multiple.test/0
16:59:22 make: *** [test] Killed
16:59:22 make: Leaving directory `/scrap/jenkins/workspace/hpc-ucx-pr-3/label/hpc-test-node3/worker/0/build-test/test/gtest'
16:59:22 
Build step 'Execute shell' marked build as failure
```

**Comment by alinask** [comment]
```
20:12:15 [----------] 1 test from ud/test_ucp_peer_failure_zcopy
20:12:15 [ RUN      ] ud/test_ucp_peer_failure_zcopy.status_after_error/0
20:12:19 /scrap/jenkins/workspace/gh-ucx-pr/label/ARM_test_node/worker/3/contrib/../test/gtest/ucp/test_ucp_peer_failure.cc:121: Failure
20:12:19 Value of: req->completed
20:12:19   Actual: false
20:12:19 Expected: true
20:12:19 /scrap/jenkins/workspace/gh-ucx-pr/label/ARM_test_node/worker/3/contrib/../test/gtest/ucp/test_ucp_peer_failure.cc:122: Failure
20:12:19 Value of: req->status
20:12:19   Actual: 0
20:12:19 Expected: m_err_status
20:12:19 Which is: -80
20:12:19 /scrap/jenkins/workspace/gh-ucx-pr/label/ARM_test_node/worker/3/contrib/../test/gtest/common/test.cc:228: Failure
20:12:19 Failed
20:12:19 Got 1 warnings during the test
20:12:19 [  FAILED  ] ud/test_ucp_peer_failure_zcopy.status_after_error/0, where GetParam() = \ud (4609 ms)
20:12:19 [----------] 1 test from ud/test_ucp_peer_failure_zcopy (4609 ms total)
```

**Comment by alinask** [comment]
```
6:41:35 [----------] 1 test from udx/test_ucp_peer_failure_zcopy
16:41:35 [ RUN      ] udx/test_ucp_peer_failure_zcopy.status_after_error/0
16:41:57 /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-test-node3/worker/0/contrib/../test/gtest/ucp/test_ucp_peer_failure.cc:121: Failure
16:41:57 Value of: req->completed
16:41:57   Actual: false
16:41:57 Expected: true
16:41:57 /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-test-node3/worker/0/contrib/../test/gtest/ucp/test_ucp_peer_failure.cc:122: Failure
16:41:57 Value of: req->status
16:41:57   Actual: 0
16:41:57 Expected: m_err_status
16:41:57 Which is: -80
16:41:58 /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-test-node3/worker/0/contrib/../test/gtest/common/test.cc:228: Failure
16:41:58 Failed
16:41:58 Got 1 warnings during the test
16:41:58 
[  FAILED  ] udx/test_ucp_peer_failure_zcopy.status_after_error/0, where GetParam() = \ud_mlx5 (22694 ms)
16:41:58 [----------] 1 test from udx/test_ucp_peer_failure_zcopy (22698 ms total)
```

**Comment by yosefe** [comment]
fixed by #1829
